config/cloud.cfg.d/10_vyos.cfg

```diff
@@ -20,11 +20,11 @@ cloud_init_modules:
 # The modules that run in the 'config' stage
 cloud_config_modules:
   - vyos
+  - write_files
   - vyos_userdata
 
 # The modules that run in the 'final' stage
-cloud_final_modules:
-  - runcmd
```

Comment:
I also removed the `runcmd` module. The module only writes a script file that will not be executed unless the `scripts_user` module is also enabled.